.travis.yml

```diff
@@ -1,22 +1,37 @@
 language: scala
-matrix:
+
+stages:
+  - name: test
+  - name: release
+    if: (tag IS present) AND NOT fork
```

Comment:
So `version.sbt` is no more and we only rely on tags? Are we suppose to tag manually for release and push?

Reply:
That's right. You would create a tag starting with `v` and the plugin is supposed to pick it up and publish to Bintray.